src/main/java/frc/robot/OI.java

```diff
@@ -14,4 +22,23 @@ public double getRightY() {
     public double getLeftX() {
         return left.getX();
     }
+
+    public OI() {
+        IntakeRoller intakeRoller = IntakeRoller.getInstance();
+        IntakePlacer intakePlacer = IntakePlacer.getInstance();
+        Button intake = controller.getRBButton();
+        intake.whenPressed(new IntakeCargo());
+        Button rollOut = controller.getLeftButton();
+        rollOut.whileHeld(new MoveGenericSubsystem(intakeRoller, IntakeRoller.MAX_SPEED));
+        Button rollIn = controller.getRightButton();
+        rollIn.whileHeld(new MoveGenericSubsystem(intakeRoller, IntakeRoller.MIN_SPEED));
+        Button intakePlacerUp = controller.getUpButton();
+        intakePlacerUp.whileHeld(new MoveGenericSubsystem(intakePlacer, IntakePlacer.MAX_SPEED).
+                andThen(new InstantCommand(() -> Robot.shouldClose = false)));
+        Button intakePlacerDown = controller.getDownButton();
+        intakePlacerDown.whileHeld(new MoveGenericSubsystem(intakePlacer, IntakePlacer.MAX_SPEED).
```

Comment:
might change after yotam changes his implementation

Reply:
We would do conflicts solving then